### DISCUSSION TITLE
Saving integers as Map

### DISCUSSION THREAD

**Discussion by IngridJSJ** [Spine-project/Spine-Toolbox#1752] [Support]
Hello!
I am trying to save a map of integers from python code.
Nevertheless while saving,  SpineDB only  me save the values as strings. 
Otherwise it throws the error: 

  File "c:\users\isanchezjimene\documents\spine-toolbox\src\spinedb-api\spinedb_api\parameter_value.py", line 594, in _map_values_from_database
    raise ParameterValueFormatError(f'Unsupported value type for Map: "{type(value).__name__}".')
spinedb_api.exception.ParameterValueFormatError: Unsupported value type for Map: "int".

I saved the values as strings but this is more time consuming, to make strings and then back to integers
    def stage_financial_results(self, financialreports):
        for fr in financialreports:
            object_name = fr.name
            self.stage_object(self.financial_reports_object_classname, object_name)
            self.stage_object_parameter_values(self.financial_reports_object_classname, object_name,
                                               [
                                                ('spotMarketRevenue', Map([str(fr.tick)], [str(fr.spotMarketRevenue)])),
                                                ('irr', Map([str(fr.tick)], [str(fr.irr)])),
                                                ],
                                               '0')
Any ideas of how to save the values as integers? 
Thank you very much!

**Comment by soininen** [comment]
`spinedb_api` doesn't currently support integers "officially". Would it be OK for your use case to cast the values to floating point numbers using `float()`?
